### PR TITLE
Scale uprising nails + clean up Uprising rules

### LINKED ISSUE #24: Balancing rallies + uprising rules
We had discussed potential balance changes around rallies and uprising. For example, in a large game rallying can be overpowered. It can allow Lads to complete an entire uprising nail in less than one turn rotation (making it very difficult for Lords to make any progress). There were a few potential balance changes we'd considered:

1. Scale up the number of uprising nails with number of players. For example, the number of uprising nails required for an uprising could be `ceiling(n/7)`
2. We could make accumulating rallies progressively more difficult depending on how many are stacked up. E.g., increasing the Rally count may only require 1 person for the first 2 rallies, but require 2 above that.
3. Lords can perform extra flips during their turn to remove rallies

There are other ways we could make this more balanced. Open to suggestions. Any thoughts?

### PR DESCRIPTION
**Pull request by seanKenkeremath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,14 +222,14 @@ If a Lad strikes the final center nail and finishes it, that player is removed f
...
+As mentioned in the Uprising rules, a Lord may call "Uprising" at the start of their turn and perform an Uprising flip similar to how a Lad would (including the extra flip). If this does not result in the final Uprising nail being finished, or the necessary number of flips are not performed, that Lord is instantly demoted back to Lad and must start a new nail.
 
 ### IX.E - A Lad illegally strikes an Uprising nail
 
-If a Lad does not call "Uprising" or does not perform the necessary number of flips, the Uprising nail is reset along with that Lad's nail (they must hammer their current starting nail into the stump if it is not already finished before beginning a new one).
+If a Lad does not call "Uprising" or does not perform the necessary number of flips, all Uprising nails are reset along with that Lad's nail (they must hammer their current starting nail into the stump if it is not already finished before beginning a new one).
```
Comment (austenlux) [inline README.md]: >they must hammer their current starting nail

maybe also add remove as on option? Or also just leave as is if it's already a brand new nail that has never been hit?

File: README.md [hunk 2]
```diff
@@ -153,25 +153,23 @@ If a Lord drops their hammer during the Flip phase they are instantly demoted to
...
+During an Uprising turn, the active player must perform one flip more than they normally would for a given number of strikes. In other words, the amount of strikes a player gets during an Uprising turn is `f - 1` where `f` is the number of flips performed (e.g., a double flip grants 1 strike). The first strike performed during an Uprising turn must be on an Uprising nail. Any other nails that are hit before a succesful strike on an Uprising nail are considered invalid and will incur penalties as outlined in the [Penalties](#ix-penalties) section. If the active player finishes the final Uprising nail, the Uprising is considered complete and all Lords are immediately demoted to Lads and must begin new starting nails. If that player has already finished their starting nail (i.e., the House of Lords was previously full), they instantly ascend to Lord. The Uprising is now no longer active and a new one may be started by future Lads.
```
Comment (austenlux) [inline README.md]: The syntax to link to a header requires replacing all spaces with dashes, so it has to be:

`[Penalties](#ix---penalties)`

File: README.md [hunk 3]
```diff
@@ -153,25 +153,23 @@ If a Lord drops their hammer during the Flip phase they are instantly demoted to
...
+A Lord may also call "Uprising" at the beginning of their turn before flipping the hammer. If they do, the current turn counts as an Uprising and Uprising rules apply for the remainder of the turn. The Uprising flip and subsequent strikes must be performed exactly as they would for a Lad. If these strikes finish the final Uprising nail the Uprising is considered complete and all other Lords are immediately demoted to Lads and must begin new starting nails. The active player retains their seat in the House of Lords. Drinks may be given out for each strike as they would for normal hits on a center nail as a Lord. If the strikes do not finish the final Uprising nail, or strikes are made without successfully completing the Uprising flip, penalties are incurred as mentioned in the [Penalties](#ix-penalties) section.
```
Comment (austenlux) [inline README.md]: Same header link syntax update needed here:

`[Penalties](#ix---penalties)`

**Review by austenlux — CHANGES_REQUESTED** [review]
👀 v good. Just some minor tweaks and a question.

Also note that the Penalties header link in the TOC also has the wrong syntax on this branch. But I fixed that in my Sparks branch, so no need to fix it here.

File: README.md (hunk 2 above)
Comment (austenlux) [reply README.md]: >the Uprising is considered complete

also I vote for the clearly far superior term of "squanched" 😂

File: README.md (hunk 1 above)
Comment (seanKenkeremath) [reply README.md]: Updated it to point to the Resetting Nails section which explains better

**Review by austenlux — APPROVED** [review]

**Pull request merged**